react-big-calendar/index.d.ts

```diff
@@ -26,10 +26,10 @@ declare module 'react-big-calendar' {
         step?: number;
         rtl?: boolean;
         eventPropGetter?: (event: Object, start: stringOrDate, end: stringOrDate, isSelected: boolean) => void;
-        titleAccessor?: string;
-        allDayAccessor?: boolean;
-        startAccessor?: stringOrDate;
-        endAccessor?: stringOrDate;
+        titleAccessor?: string | ((row: Object) => string);
+        allDayAccessor?: string | ((row: Object) => boolean);
```

Comment:
Hi Piotr.  I have changed boolean to string here because we're not trying to type guard the resultant type, but instead the provided parameter type, which will be either a _string_ property name of the current event object or a _function_ that should return a boolean.

Some use cases:
  allDayAccessor: 'someAllDayFieldName'   <-  This has to be a string; eventObj[string]  eventObj[Date] will not function as intended.
  allDayAccessor: (row:any) => row.isAllDay  <-  This function must return a boolean.